kfs-core/src/main/java/edu/arizona/kfs/coa/document/validation/impl/OrgReviewRoleRule.java

```diff
@@ -29,6 +30,51 @@ protected boolean validateFundGroupSubFundGroupCode(OrgReviewRole orr) {
 	}
 	
 	@Override
+    protected boolean validateRoleMember(OrgReviewRole orr){
+        boolean valid = true;
+        if(StringUtils.isNotEmpty(orr.getPrincipalMemberPrincipalName())){
+            if (orr.getPerson() == null) {
+                putFieldError(OrgReviewRole.PRINCIPAL_NAME_FIELD_NAME, "error.document.orgReview.invalidPrincipal"
+                        , getDataDictionaryService().getAttributeLabel(OrgReviewRole.class, OrgReviewRole.PRINCIPAL_NAME_FIELD_NAME) );
+                valid = false;
+            }
+        }
+        if(StringUtils.isNotEmpty(orr.getRoleMemberRoleName())){
+            if ( StringUtils.equals( KFSConstants.SysKimApiConstants.ACCOUNTING_REVIEWER_ROLE_NAME, orr.getRoleMemberRoleName())
+                    || StringUtils.equals( KFSConstants.SysKimApiConstants.ORGANIZATION_REVIEWER_ROLE_NAME, orr.getRoleMemberRoleName() ) ) {
+                putFieldError(OrgReviewRole.ROLE_NAME_FIELD_NAME, "error.document.orgReview.recursiveRole" );
+            } else {
+                if(orr.getRole() == null){
+                    putFieldError(OrgReviewRole.ROLE_NAME_FIELD_NAME, "error.document.orgReview.invalidRole"
+                            , new String[] {
+                                      getDataDictionaryService().getAttributeLabel(OrgReviewRole.class, OrgReviewRole.ROLE_NAME_FIELD_NAME)
+                                    , getDataDictionaryService().getAttributeLabel(OrgReviewRole.class, OrgReviewRole.ROLE_NAME_FIELD_NAMESPACE_CODE)
+                                    } );
+                    valid = false;
+                }
+            }
+        }
+        if(StringUtils.isNotEmpty(orr.getGroupMemberGroupName())){
+            if( orr.getGroup() == null ){
+                putFieldError(OrgReviewRole.GROUP_NAME_FIELD_NAME, "error.document.orgReview.invalidGroup"
+                        , new String[] {
+                                  getDataDictionaryService().getAttributeLabel(OrgReviewRole.class, OrgReviewRole.GROUP_NAME_FIELD_NAME)
+                                , getDataDictionaryService().getAttributeLabel(OrgReviewRole.class, OrgReviewRole.GROUP_NAME_FIELD_NAMESPACE_CODE)
+                                } );
+                valid = false;
+            }
+        }
+        for(String roleName : orr.getRoleNamesToConsider()) {
+        	if(StringUtils.equals(roleName, KFSConstants.SysKimApiConstants.ORGANIZATION_FUND_REVIEWER_ROLE_NAME) && StringUtils.isBlank(orr.getFundGroupCode()) && StringUtils.isBlank(orr.getSubFundGroupCode())) {
+        		putFieldError(KfsKimAttributes.SUB_FUND_GROUP_CODE, "error.member.fundgroup.subfundgroup.onerequired");
+        		valid = false;
+        	}
+        }
```

Comment:
I think I put it there since that is where it was in KFS3, but you are correct. It could be added to the method validateFundGroupSubFundGroupCode.  I will move it and retest.